src/DTO/Mapper/QuapQuestionnaireMapper.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace App\DTO\Mapper;
+
+use App\DTO\Model\AspectDTO;
+use App\Entity\Questionnaire;
+use DateTime;
+
+class QuapQuestionnaireMapper { // this name might not be all too fitting
```

Comment:
Why do you add the Quap prefix to this Mapper? You didn't add it to all the others.